data/json/recipes/armor/arms.json

```diff
@@ -66,7 +74,11 @@
     "time": "260 m",
     "book_learn": [ [ "textbook_tailor", 3 ], [ "manual_tailor", 4 ], [ "tailor_portfolio", 3 ] ],
     "byproducts": [ [ "scrap_neoprene", 43 ] ],
-    "using": [ [ "plastic_molding", 2 ], [ "tailoring_neoprene_patchwork", 2 ] ]
+    "using": [ [ "plastic_molding", 2 ], [ "tailoring_neoprene_patchwork", 2 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_polymerworking", "required": false, "time_multiplier": 3 },
+      { "proficiency": "prof_plasticworking", "required": false, "time_multiplier": 2 }
```

Comment:
Since this is one that probably mostly uses hard plastic I'd just swap the multipliers on these